config/default/system.mail.yml

```diff
@@ -3,3 +3,10 @@ _core:
 interface:
   default: php_mail
   webform: webform_php_mail
+mailer_dsn:
+  scheme: sendmail
```

Comment:
Wondering if this is the correct scheme. I think this is based on updates to the core mail system.

Reply:
My understanding is that this is correct for PROD, but might have an effect on our non-production tweak in global settings?

Reply:
Just double-checking.